src/NimBLECharacteristic.cpp

```diff
@@ -1,4 +1,9 @@
 /*
+ * NimBLECharacteristic.cpp
+ *
+ *  Created: on Aug 27, 2022
+ *      Author Gluhovsky
+ *
```

Comment:
Please don't change this like that. You can add a "modified" section with your signature if you like.